e2e/test/scenarios/dashboard-filters/shared/dashboard-filters-text-category.js

```diff
@@ -1,26 +1,67 @@
-export const DASHBOARD_TEXT_FILTERS = {
-  Is: {
+export const DASHBOARD_TEXT_FILTERS = [
+  {
+    operator: "Is",
+    single: true,
```

Comment:
All operators except `Is` and `Is not` have tests for both `single: true` and `single: false`.
Any reason for `Is` and `Is not` not having tests with `single: false`?

Reply:
`Is` and `Is not` are using a helper that's too rigid and will not work with multiple values.
This is the consequence of trying to abstract too much.

I didn't want to change the underlying logic of that helper since it's very likely that these tests will have to be update and/or broken down soon.